api/json-schema/schema.json

```diff
@@ -19074,13 +19074,13 @@
           "format": "int64",
           "type": "integer"
         },
-        "targetBufferAvailability": {
-          "description": "TargetBufferAvailability is used to define the target percentage of the buffer availability. A valid and meaningful value should be less than the BufferUsageLimit defined in the Edge spec (or Pipeline spec), for example, 50. It only applies to UDF and Sink vertices because only they have buffers to read.",
+        "targetBufferUsage": {
+          "description": "TargetBufferUsage is used to define the target percentage of the buffer availability. A valid and meaningful value should be less than the PartitionUsageLimit defined in the Edge spec (or Pipeline spec), for example, 50. It only applies to UDF and Sink vertices because only they have buffers to read. Deprecated: use targetBufferAvailability instead. Will be removed in v0.9",
```

Comment:
Do you mean `targetBufferUsage` will be deprecated and use `targetPartitionAvailability`? Or `targetBufferAvailability`. If the later, I don't see it defined after the change?